docs/version-2-upgrade.md

```diff
@@ -0,0 +1,385 @@
+
+For example, given this previous configuration:
+
+```terraform
+module "runtime_connector" {
+  source  = "symopsio/runtime-connector/aws"
+  version = "~> 1.0"
+
+  environment = var.environment
+}
+```
+
+An updated configuration would be:
+
+```terraform
+module "runtime_connector" {
+  source  = "symopsio/runtime-connector/aws"
+  version = "~> 2.0"
+
+  environment = var.environment
+}
+```
+
+Optionally, you may choose to use `moved` configuration blocks to migrate your Terraform state, instead of replacing the existing AWS IAM Role and Policy with a new one:
+```terraform
+# The following blocks may be removed after applying the updated configuration
+moved {
+  from = module.runtime_connector.aws_iam_role.this
+  to   = module.runtime_connector.aws_iam_role.sym_runtime_connector_role
+}
+
+moved {
+  from = module.runtime_connector.aws_iam_role_policy_attachment.assume_roles_attach
+  to   = module.runtime_connector.aws_iam_role_policy_attachment.attach_assume_roles
+}
+```
+
+## Removed Inputs: `addons` and `addon_params`
+The `addons` and `addon_params` inputs have been removed as of `runtime_connector` version 2.0.
+
+Instead of passing in a list of addon module names, these modules should now be declared directly in your configuration. The addons should be refactored as follows:
+- `aws/secretsmgr`: `symopsio/secretsmgr-addon/aws@1.1` ([View in Terraform registry](https://registry.terraform.io/modules/symopsio/secretsmgr-addon/aws/latest))
+- `aws/kinesis-firehose`: `symopsio/kinesis-firehose-addon/aws@1.1` ([View in Terraform registry](https://registry.terraform.io/modules/symopsio/kinesis-firehose-addon/aws/latest))
+- `aws/kinesis-data-stream`: `symopsio/kinesis-data-stream-addon/aws@1.1` ([View in Terraform registry](https://registry.terraform.io/modules/symopsio/kinesis-data-stream-addon/aws/latest))
+
+### Refactoring the `aws/secretsmgr` Addon
+If `aws/secretsmgr` is declared as an addon, update your configuration to add the `symopsio/secretsmgr-addon/aws` module explicitly.
+
+For example, given this previous configuration:
+```terraform
+module "runtime_connector" {
+  source  = "symopsio/runtime-connector/aws"
+  version = "~> 1.0"
+
+  environment = var.environment
+  
+  addons = ["aws/secretsmgr"]
+}
+```
+
+An updated configuration would be:
+
+```terraform
+module "runtime_connector" {
+  source  = "symopsio/runtime-connector/aws"
+  version = "~> 2.0"
+
+  environment = var.environment
+}
+
+module "secrets_manager_access" {
+  source  = "symopsio/secretsmgr-addon/aws"
+  version = "~> 1.1"
+
+  environment   = var.environment
+  iam_role_name = module.runtime_connector.sym_runtime_connector_role.name
+}
+```
+
+Optionally, you may choose to use `moved` configuration blocks to migrate your Terraform state, instead of replacing the existing IAM policies with new ones:
```

Comment:
```suggestion
Optionally, you may also use `moved` configuration blocks to migrate your Terraform state, instead of replacing the existing IAM policies with new ones:
```